### DISCUSSION TITLE
4.2 Confusion

### DISCUSSION THREAD

**Discussion by Connor-McCloskey** [utopia-rise/fmod-gdextension#196] [General]
Hi there!

I'm a little confused about whether this project supports Godot 4.2 or not, so I figured I'd ask! It seems to going off of the title of the latest release, but when I try to point my project to my banks, it never seems to display them. Without more information, I didn't want to file a bug report, and it seems omnipresent enough that it seems like it's either a big bug or a symptom of the implementation for 4.2 just not being there yet :)

**Comment by piiertho** [comment]
Hello !
Yes, latest release is for 4.2.
Do you have any log so we can help you ?

**Reply by Connor-McCloskey** [reply]
Oh, fascinating! Here's what I can provide at the moment then.
- I am specifically working off 4.2.1 if that makes a difference
- I have the plugin enabled, and I added the folder to where my banks exist within the Godot project
- However, when I open the Fmod Explorer, it is completely blank. Same with adding a bank loader to the scene.
- I get a few warnings when I open the project, which I'm not sure are relevant but I've added them below just in case

![blank_fmod_explorer](https://github.com/utopia-rise/fmod-gdextension/assets/65367191/80e429cc-2ceb-45e5-aaf7-a29a9d30a89b)
[copy_Fmod_Log_items.txt](https://github.com/utopia-rise/fmod-gdextension/files/14436794/copy_Fmod_Log_items.txt)

**Reply by Connor-McCloskey** [reply]
Aha, I seem to have figured it out. Our naming conventions differed from what the plugin required - for example, we had our master bank named "Master Bank.bank" and not just "Master.bank."

**Reply by piiertho** [reply]
Regarding the log you sent it seems to miss master string bank. Can you add it ?

**Reply by Connor-McCloskey** [reply]
Is there a better place to find documentation on how to use this plugin? It seems like a great piece of software, but some of the workflow and requirements (like this) aren't evident.

**Reply by Connor-McCloskey** [reply]
@piiertho Yep! I've added it now in the required format. We had a string bank, it just wasn't named in that exact way so it couldn't find the file to load. I'll continue playing around with the plugin now that I'm unblocked there, I'd still love to find more documentation if it exists though!

**Reply by piiertho** [reply]
We have a doc site here: https://fmod-gdextension.readthedocs.io/en/latest/

Be aware that `2 - Add Fmod libraries to appropriate folder` is not required anymore. (We forgot to remove it)

Please report if something isn't clear so we can improve it.

**Reply by Connor-McCloskey** [reply]
Will do, thank you!

**Comment by Flo-Roeder** [comment]
hey guys, i have a problem with getting fmod work in godot 4.2.1,
there must be something - properbly simple - i am just not seeing...

- addon is installed and set up.
- explorer and emitter show events and i can select it
- i also can preview the events in the explorer menu

but as soon as i start a scene with an emitter it immediately closes
- no errors
- nothing
- instant close

after playing around a bit i would say it is something with loading the events, because it only cancels the scene if "autoplay" or "preload event" is checked.

can someone help please

**Reply by Flo-Roeder** [reply]
ok nvm.
i just had to setup the FMODBankLoader 😶‍🌫️
